<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1313</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="imgSrc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="procBtn">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>실행</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadBtn">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>10</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>이미지 로드</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgDst">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="filePath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1031</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>원본 이미지</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>600</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>결과 이미지</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>650</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>THRESH_BINARY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>THRESH_BINARY_INV</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>THRESH_TRUNC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>THRESH_TOZERO</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>THRESH_TOZERO_INV</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>THRESH_OTSU</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>이진화 방법</string>
   </property>
  </widget>
  <widget class="QSlider" name="trackbar">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>680</y>
     <width>591</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_threshold">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>670</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>임계치</string>
   </property>
  </widget>
  <widget class="QSlider" name="maxValueSlider">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>640</y>
     <width>581</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_threshold_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>630</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>맥스</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
